rfcs/2019-v3-dom-props.md

```diff
@@ -50,7 +50,7 @@ This can be enabled by supporting explicit props for styling in the API for each
 <Button margin="size-250">A button</Button>
 ```
 
-The full list of supported style props can be seen [here](https://github.com/adobe-private/react-spectrum-v3/blob/db1b2716cbe934323866a8bb5391284e0659e569/packages/%40react-spectrum/view/src/types.ts#L4).
+The full list of supported style props can be seen [here](https://github.com/adobe/react-spectrum/blob/db1b2716cbe934323866a8bb5391284e0659e569/packages/%40react-spectrum/view/src/types.ts#L4).
```

Comment:
I can replace this with https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-types/shared/src/style.d.ts right?

Reply:
that's probably a good idea